mds/migrations/0017_auto_20190328_1619.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.1.7 on 2019-03-28 16:19
+
+from django.db import migrations
+import mds.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mds', '0016_agency_api_configuration'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='device',
+            name='category',
+            field=mds.models.UnboundedCharField(choices=[('bicycle', 'Bicycle'), ('scooter', 'Scooter'), ('car', 'Car')]),
```

Comment:
Shouldn't we see both `bike` and `bicycle` in this intermediate release?

Reply:
I thought we were changing directly, that's what I was told in my ticket